### PR TITLE
Feat: enable user see a summary of user data

### PR DESCRIPTION
**Pull request by khalifan-kfan**
# Description

Enable user see a summary of user data

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Trello Ticket ID

https://trello.com/c/i1kvPTlL

## How Can This Be Tested?

Run this branch and try to fetch any profiles's summary  on route 
http://localhost:50000/users/profile_summary/<id>

### CODE REVIEW

File: app/controllers/users.py [hunk 1]
```diff
@@ -1264,3 +1266,47 @@ def get(self, user_id):
...
+        """
+        Returns metrics about a user's following information and their resource counts.
+        """
+        try:
+            user = User.get_by_id(user_id)
+
+            if not user:
+                return dict(status='fail', message='User not found'), 404
+
+            print('hdf')
```
Comment (LanternNassi) [inline app/controllers/users.py]:
```suggestion
```

File: app/controllers/users.py [hunk 2]
```diff
@@ -1264,3 +1266,47 @@ def get(self, user_id):
...
+
+            user_data = {
+                'projects_count': len(user.projects),
+                'following_count': user.followed.count(),
+                'follower_count': user.followers.count(),
+                'followed_projects_count': len(user.followed_projects),
+                'projects_followers_count': count_of_projects_followers,
+                'apps_count': project_app_counts.total()
+            }
+            print(user_data)
```
Comment (LanternNassi) [inline app/controllers/users.py]:
```suggestion
```

**Review by LanternNassi — DISMISSED** [review]
LGTM

**Review by Mubangizi — CHANGES_REQUESTED** [review]
Hey @khalifan-kfan, Great work here. But I think we already have an endpoint for this and its under `GET` for `UserDetailView(Resource):`. So if you want to add more info just update what is returned on that route

**Comment by khalifan-kfan** [comment]
Alright @Mubangizi  and @LanternNassi, thanks for the feedback, this is going to be worked on

**Comment by khalifan-kfan** [comment]
@Mubangizi  and @LanternNassi  this has been resolved

**Review by Mubangizi — APPROVED** [review]

**Pull request merged**